src/ducks/emergency-save.js

```diff
@@ -0,0 +1,64 @@
+/*
+Emergency Save Work In Progress
+-------------------------------
+ASM has a special problem: sometimes, when a user is working on a document for
+too long, their login session dies. When this happens, their work is saved in
+local storage, so they can continue where they left off after refreshing the
+page (and refreshing their login state).
+
+NOTE: This emergecy system does NOT care which account the user is logged in to
+when they save their work, and later resume their work. This system isn't trying
+to be smart, it's a blunt instrument acting as a safety net when the user's Save
+Progress and Panoptes Client's token auto-refresh fails.
+ */
+
+import { fetchSavedSubject, prepareForNewSubject } from './subject';
+import { setAnnotations } from './annotations';
+
+const SUBJECT_ID_KEY = 'emergency_save_subjectId';
+const ANNOTATIONS_KEY = 'emergency_save_annotations';
+
+function checkEmergencySave() {
+  const subjectId = localStorage.getItem(SUBJECT_ID_KEY);
+  const annotations = localStorage.getItem(ANNOTATIONS_KEY);
+
+  return !!subjectId && !!annotations;
```

Comment:
Should there be a check in this function, or before, that a user is logged in? Ex. Lets say a user must refresh his/her page because the token is expired and they are logged out. When the page reloads (on the classifier), the `DialogOfContinuation` will appear allowing a user to load the saved annotation without necessarily logging on again.

Reply:
Ex. The `DialogOfFailure` says "you may need to reload the page and log in again," but the page can be reloaded and the emergency classification can be retrieved again without logging back in. 

Note: this is likely coming from line 250 of the subject duck. However, I can understand this is the functionality you want with a non-logged in user, but it might throw off the logged in user who begins a classification logged in and continues as a guest.

Reply:
Will, you are correct - the emergency save does NOT care if the user is logged in, and it's entirely possible for a non-logged-in User X to continue a logged-in User Y's work (if that User Y was booted off by an expired token). It's designed this way to favour the user experience, allowing them to see their work even if they 'forget' to re-login.

This was buried in my comments above, sorry:
https://github.com/zooniverse/anti-slavery-manuscripts/pull/251#issuecomment-368192732
> The emergency save is a blunt instrument acting as a safety net. It does not, for example, care if you're logged in to the same account when it restores your emergency save. So for example, if you transcribed half the page as zootester1, then encountered a session error, reloaded, was prompted to login, then logged in as zootester2 (or continue as a non-logged-in user), and continued, it will not care and just load the saved progress anyway.